optimizely/helpers/types.py

```diff
@@ -76,3 +76,25 @@ class RolloutDict(TypedDict):
     '''Rollout dict from parsed datafile json.'''
     id: str
     experiments: list[ExperimentDict]
+
+
+class FeatureFlagDict(BaseDict):
+    '''Feature flag dict from parsed datafile json.'''
+    rolloutId: str
+    variables: list[VariableDict]
+    experimentIds: list[str]
+
+
+class GroupDict(TypedDict):
```

Comment:
Is it possible instead of directly inheriting TypedDict, create a BaseEntity ? where we don't have any Id,Key and then we have BaseDict inherity from BaseEntity. 
Use BaseEntity instead of TypedEntity where needed.

Reply:
Yea that should work